From 72c555d49e11ccb0077a5a5f5b20309c2b7262bf Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Fri, 26 Jul 2013 13:59:49 +0800
Subject: [PATCH 09/45] ENGR00271393 playback quit if subtitle on and loop
 play with cactus player

local reference not released, make reference table overflow

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 media/jni/android_media_MediaPlayer.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/jni/android_media_MediaPlayer.cpp b/media/jni/android_media_MediaPlayer.cpp
index d8041f4..cfda218 100644
--- a/media/jni/android_media_MediaPlayer.cpp
+++ b/media/jni/android_media_MediaPlayer.cpp
@@ -125,6 +125,7 @@ void JNIMediaPlayerListener::notify(int msg, int ext1, int ext2, const Parcel *o
             env->CallStaticVoidMethod(mClass, fields.post_event, mObject,
                     msg, ext1, ext2, jParcel);
             env->DeleteLocalRef(jParcel);
+            jParcel = NULL;
         }
     } else {
         env->CallStaticVoidMethod(mClass, fields.post_event, mObject,
-- 
1.8.0

